definitions/tools/mark_duplicates_and_sort.cwl

```diff
@@ -40,11 +40,7 @@ inputs:
         default: "queryname"
         inputBinding:
             position: 5
-    output_name:
```

Comment:
this input will need to be added back in as the output `sorted_bam` references it. I believe it'll only throw an error at runtime because `cwltool --validate` will not catch it